docs/writing-tests/index.md

```diff
@@ -228,3 +228,72 @@ Note: When importing a group, its optional children can be omitted:
 ```ruby
 group from: :us_core_patient, exclude_optional: true
 ```
+
+## Importing From Other Test Kits
+
+Inferno Test Kits may include other Test Kits as a dependency. This allows you to
+reuse existing tests, test groups or test suites as part of your own tests.  This is helpful
+if your testing requires common functionality already covered by existing Test Kits, such as
+[SMART App Launch](https://github.com/inferno-framework/smart-app-launch-test-kit) for testing 
+connecting to a FHIR Server, and [Bulk Data Access Test Kit](https://github.com/inferno-framework/bulk-data-test-kit) for
+exporting bulk data from a FHIR Server.
+
+To start, add each Test Kit you want to import as a runtime dependency in the `.gemspec` 
+file of your test kit, as you would for additional Ruby gems.
+
+```ruby
+Gem::Specification.new do |spec|
+  spec.name          = 'my_custom_test_kit'
```

Comment:
I'd cut out all of the stuff in here that isn't a `spec.add_runtime_dependency` so that the focus is really on the changes to add a test kit dependency.

Reply:
Updated the example